charts/metal3/0.2.0/charts/ironic/values.yaml

```diff
@@ -88,15 +87,30 @@ podAnnotations: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
+requireProvisioningIP: true
```

Comment:
I don't think this should be configurable - the only reason it was a variable in the previous service list was that not all containers require `PROVISIONING_IP` - but for those that do, we should always pass either the provisioningIP, or when no provisioning network is enabled the hostIP.

Reply:
I left it here, because in Sylva's use-case it needs to be set to false, since there we do not provide `PROVISIONING_IP`, but `PROVISIONING_INTERFACE`. When providing provisioning interface and this property is not set to false the ironic pod failed (at least on Sylva's clusters). That is why it is left configurable here.

Reply:
Oh that is a good point!  The reason we pass `hostIP` as `PROVISIONING_IP` is the [ironic container waits](https://github.com/metal3-io/ironic-image/blob/main/scripts/ironic-common.sh#L34) for either `PROVISIONING_INTERFACE` to come up, or for (any) interface to come up with `PROVISIONING_IP` - when there is no dedicated provisioning network we don't need to wait, as Ironic will just use the `hostIP`

So I still think we can remove this value, but adjust the conditionals in the deployment template to only set `PROVISIONING_IP` to `hostIP` when neither `provisioningIP` or `provisioningInterface` are set?